ui/components/multichain/import-account/private-key.js

```diff
@@ -28,11 +30,10 @@ export default function PrivateKeyImportView({
 
   return (
     <>
-      <FormTextField
+      <FormPasswordField
         id="private-key-box"
         size={TextFieldSize.Lg}
         autoFocus
-        type={TextFieldType.Password}
```

Comment:
You can create a `FormPasswordField` if you like but you could also achieve this by including the `ShowHideToggle` button using the `FormTextField` => `textFieldProps` => `endAccessory` prop

```jsx
import {
  FormTextField,
  TextFieldSize,
  TextFieldType,
} from '../../component-library';

const [showPrivateKey, setShowPrivateKey] = useState(false);

<FormTextField
  id="private-key-box"
  size={TextFieldSize.Lg}
  autoFocus
  type={showPrivateKey ? TextFieldType.Text : TextFieldType.Password}
  textFieldProps={{
    endAccessory: (
      <ShowHideToggle
        shown={showPrivateKey}
        id="show-hide-private-key"
        title={t('privateKeyShow')}
        ariaLabelShown={t('privateKeyShown')}
        ariaLabelHidden={t('privateKeyHidden')}
        onChange={() => setShowPrivateKey(!showPrivateKey)}
      />
    ),
  }}
```


https://github.com/MetaMask/metamask-extension/assets/8112138/060db4aa-9e7e-4eb8-a981-ba9d6c57e953

Reply:
Thank you very much @georgewrmarshall , i will use this props to redo the `private-key.js` and remove the extra `form-password-field` to reduce the complexity. :)

Reply:
hi, @georgewrmarshall i have updated the `private-key.js` to use `form-text-field` with `endAccessory` attribute, please when you have time, review again the file and `resolve` this conversation.  thank you very much.